android-sdk/src/main/java/com/mobilecoin/lib/TxOutStore.java

```diff
@@ -46,6 +44,8 @@
 
     // A map of nonce -> Seed.
     private HashMap<Integer, FogSeed> seeds;
+
+    private Set<Long> decommissionedIngestInvocationIds;
```

Comment:
Similar to my other comment, it looks like there is work that needs to happen to persist this and deal with versions that don't have it.

Reply:
Same as above.